airflow/config_templates/airflow_local_settings.py

```diff
@@ -184,6 +184,9 @@
 
         DEFAULT_LOGGING_CONFIG['handlers'].update(GCS_REMOTE_HANDLERS)
     elif REMOTE_BASE_LOG_FOLDER.startswith('wasb'):
+
+        # If you use URI scheme, the parameter `wasb_container` is redundancy.
```

Comment:
Should exception be fired if such redundancy detected?

Reply:
OK, I have added the check.